src/geocel/detail/PolygonUtils.hh

```diff
@@ -0,0 +1,126 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file geocel/detail/LengthUnits.hh
+//! \brief NOTE: only use inside geocel; prefer celeritas/Units.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+//#define CELER_ICRT inline constexpr real_type
+#include <iostream>
+#include <vector>
+#include <corecel/cont/Array.hh>
+#include <corecel/math/ArrayUtils.hh>
+#include <orange/OrangeTypes.hh>
+
+namespace celeritas
+{
+namespace geoutils
+{
+
+template<typename T>
+using Real2 = Array<T, 2>;
+template<typename T>
+using VecReal2 = std::vector<Real2<T>>;
+template<typename T>
+using Real3 = Array<T, 3>;
+template<typename T>
+using VecReal3 = std::vector<Real3<T>>;
```

Comment:
For a template `T` these aren't `real`s: it would have to be `real_type`. Perhaps it would be best to remove thse to avoid conflicting with the definition of `Real3` elsewhere.